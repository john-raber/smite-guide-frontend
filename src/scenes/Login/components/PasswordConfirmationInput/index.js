import React from "react";
import { FormGroup, Label, Input } from "reactstrap";

const PasswordConfirmationInput = ({ value, onChange }) => {
  return (
    <FormGroup className="PasswordConfirmationInput">
      <Label for="passwordConfirmation">Confirm Password</Label>
      <Input
        name="passwordConfirmation"
        type="password"
        placeholder="Re-Type Password"
        value={value}
        onChange={e => onChange(e.target.name, e.target.value)}
      />
    </FormGroup>
  );
};

export default PasswordConfirmationInput;
