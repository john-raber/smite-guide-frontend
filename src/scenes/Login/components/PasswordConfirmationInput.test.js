import React from "react";
import { shallow } from "enzyme";

import PasswordConfirmationInput from "./PasswordConfirmationInput/index";

describe("PasswordConfirmationInput", () => {
  it("renders correctly", () => {
    let wrapper = shallow(<PasswordConfirmationInput />);

    expect(wrapper.find(".PasswordConfirmationInput").length).toEqual(1);
  });

  it("matches its snapshot", () => {
    let wrapper = shallow(<PasswordConfirmationInput />);

    expect(wrapper).toMatchSnapshot();
  });
});
