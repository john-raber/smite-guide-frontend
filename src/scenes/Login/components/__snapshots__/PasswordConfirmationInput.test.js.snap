// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordConfirmationInput matches its snapshot 1`] = `
<FormGroup
  className="PasswordConfirmationInput"
  tag="div"
>
  <Label
    for="passwordConfirmation"
    tag="label"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    Confirm Password
  </Label>
  <Input
    name="passwordConfirmation"
    onChange={[Function]}
    placeholder="Re-Type Password"
    type="password"
  />
</FormGroup>
`;
